spec/system/teacher_interface/authentication_spec.rb

```diff
@@ -156,11 +156,11 @@ def given_i_clear_my_session
   end
 
   def when_i_visit_the_sign_up_page
```

Comment:
There's a helper in PageHelpers now that gives you this in the format `when_i_visit_the(:sign_up_page)`